assume/common/scenario_loader.py

```diff
@@ -242,7 +236,11 @@ async def load_scenario_folder_async(
         file_name="demand_units",
     )
 
-    heatpump_units = load_file(path=path, config=config, file_name="heatpumps")
+    heatpump_units = load_file(
+        path=path,
+        config=config,
+        file_name="heatpumps",
```

Comment:
```suggestion
        file_name="heatpump_units",
```

The file_name is now `heatpump_units` which is consistent with everything else